src/openloco/windows/CompanyWindow.cpp

```diff
@@ -7,6 +7,13 @@ using namespace openloco::interop;
 
 namespace openloco::ui::windows::CompanyWindow
 {
+    void OpenUnk(company_id_t companyId)
+    {
+        registers regs;
+        regs.eax = companyId;
+        call(0x0043454F, regs);
+    }
+
```

Comment:
Fwiw, this is the regular `open` function in my company window PR.

Reply:
Do you want me to rename it open then?

Reply:
No real need. I'm fine with changing it when I rebase my PR on top of this.